tests/tidb_rowid/run.sh

```diff
@@ -17,6 +17,11 @@
 
 set -eu
 
+# disable cluster index by default
+run_sql 'set @@global.tidb_enable_clustered_index = 0' || echo "tidb does not support cluster index yet, skipped!"
```

Comment:
should reset the value back to the original after finish running

Reply:
fixed